packages/sirius-web/frontend/sirius-web-application/src/navigationBar/NavigationBar.tsx

```diff
@@ -38,19 +39,22 @@ const useNavigationBarStyles = makeStyles()((theme) => ({
   },
   toolbar: {
     display: 'grid',
-    gridTemplateColumns: '1fr min-content 1fr',
+    gridTemplateColumns: '1fr 1fr 1fr',
     backgroundColor: theme.palette.navigationBar.background,
   },
   left: {
     display: 'flex',
     flexDirection: 'row',
     alignItems: 'center',
+    justifyContent: 'flex-start',
     gap: theme.spacing(1),
   },
-  link: {
+  center: {
```

Comment:
Why choose to align things this way, let consumers of the extension point do what they want. The may not want to be aligned vertically, they may not want any gap etc.

Reply:
Done.